### PR TITLE
Disable remote trigger if tests are disabled

### LINKED ISSUE #422: Remote trigger to Temurin Compliance server should not occur when nightly testing is disabled
**Expected behaviour:** Weekly runs, which have testing disabled, should not initiate the TC trigger

**Observed behaviour:** Weekly jobs are running the TC trigger during lockdown

**Any other comments:** The logs currently don't show when the trigger is being executed in the logs - ideally we should add this in since it's currently happening silently which made this harder to look into
Sample log from the [Windows weekly run which didn't run the AQA tests](https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-windows-x64-temurin/116/) but the [pipeline job did trigger TC](https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk11-pipeline/2094/console):  - it launched https://ci.eclipse.org/temurin-compliance/job/AQA_Test_Pipeline/94

Ref: functionality introduced in https://github.com/adoptium/ci-jenkins-pipelines/commit/b5ffce4bdb5c6c2702e9639c3a559a65d418ffd7

FYI @smlambert as discussed recently
FYI @sophia-guo

### PR DESCRIPTION
**Pull request by sophia-guo**
Close #422 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

File: pipelines/build/common/build_base_file.groovy [hunk 1]
```diff
@@ -908,8 +908,9 @@ class Builder implements Serializable {
                 }
             } else if (publish && release) {
                 context.println "NOT PUBLISHING RELEASE AUTOMATICALLY"
-            } else if (release ) {
+            } else if (release && enableTests) {
                 //remote trigger job https://ci.eclipse.org/temurin-compliance/job/AQA_Test_Pipeline/
+                context.echo "Enable tests: ${enableTests}"
```
Comment (karianna) [inline pipelines/build/common/build_base_file.groovy:913]:
```suggestion
                context.echo "Enable tests: ${enableTests} so we will trigger the remote JCK jobs"
```

**Review by karianna — APPROVED** [review]

**Review by sxa — APPROVED** [review]

File: pipelines/build/common/build_base_file.groovy (hunk 1 above)
Comment (sophia-guo) [reply pipelines/build/common/build_base_file.groovy]: Thanks @karianna ! That's originally added as testing code and forgot to update. Updated!

**Comment by sophia-guo** [comment]
Not sure about the Linter failure.

```
2022-10-03 20:56:51 [INFO]   Linting [JSCPD] files...
2022-10-03 20:56:51 [INFO]   ----------------------------------------------
2022-10-03 20:56:51 [INFO]   ----------------------------------------------
2022-10-03 20:56:51 [INFO]   ---------------------------
2022-10-03 20:56:51 [INFO]   File:[/github/workspace/pipelines/build/common/build_base_file.groovy]
2022-10-03 20:56:51 [INFO]    - File:[build_base_file.groovy] was linted with [jscpd] successfully
2022-10-03 20:56:51 [INFO]      - Command output:
------
Detection time:: 1ms
------
2022-10-03 20:56:53 [INFO]   ----------------------------------------------
2022-10-03 20:56:53 [INFO]   ----------------------------------------------
2022-10-03 20:56:53 [INFO]   The script has completed
2022-10-03 20:56:53 [INFO]   ----------------------------------------------
2022-10-03 20:56:53 [INFO]   ----------------------------------------------
2022-10-03 20:56:53 [INFO]   ERROR! Failed to call GitHub Status API!
2022-10-03 20:56:53 [INFO]   ERROR:[curl: (22) The requested URL returned error: 403]
2022-10-03 20:56:53 [ERROR]   ERRORS FOUND in GROOVY:[1]
```

**Comment by sophia-guo** [comment]
Linter failure see https://github.com/adoptium/ci-jenkins-pipelines/issues/425, not related with this PR.

**Pull request merged**